bloom-housing/bloom
# fix: only show lottery button if app has results

This PR addresses #4303


## Description

Only show `View lottery results` button if we actually have results - which will by default filter out duplicates that don't have results if they were not included in the lottery.

## How Can This Be Tested/Reviewed?

Ensure the button only shows on a published lottery listing if the application has results (which it won't for a duplicate you confirm and exclude from the lottery).

## Author Checklist:


## Review Process:

- Read and understand the issue
- Ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Either (1) explicitly ask a clarifying question, (2) request changes, or (3) approve the PR, even if there are very small remaining changes, if you don't need to re-review after the updates
